app/js/templates/adminTable.hbs

```diff
@@ -93,6 +93,12 @@
                     <td class="time">{{time}}</td>
                 </tr>
                 {{/if}}
+                    {{#if this.autoCompleted}}
+                <tr>
+                    <th>Auto completed:</th>
+                    <td class="autoComplete">Yes</td>
+                </tr>
+                    {{/if}}
```

Comment:
Just OCD but could move the {{#if}} & {{/if}} back 1 tab to be in line with the rest of the template file